jcommune-plugins/questions-n-answers-plugin/src/main/java/org/jtalks/jcommune/plugin/questionsandanswers/controller/QuestionsAndAnswersController.java

```diff
@@ -148,7 +148,7 @@ public String createQuestion(@Valid @ModelAttribute TopicDto topicDto, BindingRe
         Map<String, Object> data = getDefaultModel(request);
         topicDto.getTopic().setBranch(branch);
         if (result.hasErrors()) {
-            data.put(BREADCRUMB_LIST, breadcrumbBuilder.getForumBreadcrumb(branch));
+            data.put(BREADCRUMB_LIST, breadcrumbBuilder.getForumBreadcrumb(topicDto.getTopic()));
```

Comment:
This fix problem for questions. It's OK. But what about discussions and code review? The problem exist for all topic types.